aggregator/src/app/BundleRouter.ts

```diff
@@ -39,5 +39,19 @@ export default function BundleRouter(bundleService: BundleService) {
     },
   );
 
+  router.get(
+    "aggregateBundle/:subBundleHash",
```

Comment:
Thoughts on naming in this PR:

aggregateBundle: refers to the collection of bundles that were submitted with the bundle hash provided (subBundleHash). Wondering if the naming is confusing as it isn't actually an "aggregated bundle", or a bundle that was aggregated, it is a collection of bundles that were submitted along with the bundle from the hash provided.

aggregateHash: the hash given to the aggregateBundle that get's submitted on chain. Bit torn on the naming here too, as you can't actually look up the aggregate bundle with this hash as the aggregate bundle is not saved in the db, only the sub bundles (to my knowledge). It is used to group other sub bundles together, see BundleTable.ts.

subBundle: refers to the bundles saved in the DB that were submitted by a user/application. Not the aggregate bundle. `subBundleHash` is the bundle hash for said bundle.

Reply:
Given how things work now I think this is fine. Ideally, we would just have one endpoint you could provide a hash to, and it would give you the one or more userops associated with that, maybe status.

I saw in PR desc that you weren't able to find implementation of ERC-4337 way of doing it. They may not have gotten that far...